### PR TITLE
runtimetest: remove debug info 

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: cmd/runtimetest/main.go [hunk 1]
```diff
@@ -107,6 +107,8 @@ func validateGeneralProcess(spec *rspec.Spec) error {
 }
 
 func validateLinuxProcess(spec *rspec.Spec) error {
+	logrus.Debugf("validating linux process")
```
Comment (liangchenye) [inline cmd/runtimetest/main.go]: I think we can just use `logrus.Debug` .

**Comment by Mashimiao** [comment]
We don't need this. 
Before we had logrus like this, but removed. Because it is duplicate with tap's output

**Comment by wking** [comment]
On Wed, Nov 15, 2017 at 10:02:43AM +0000, Ma Shimiao wrote:
> Because it is duplicate with tap's output

More on this front in #308 and #439, both of which are currently
milestoned for v0.4.0.

**Comment by q384566678** [comment]
Finally, I think it's better to delete debug information at present.

**Comment by liangchenye** [comment]
LGTM

**Comment by Mashimiao** [comment]
LGTM

**Pull request merged**